src/app/pipes/propertydisplay.ts

```diff
@@ -10,6 +10,9 @@ export class PropertyDisplayPipe implements PipeTransform {
     if (property === 'id') {
       return property.toUpperCase();
     }
+    if (property === 'successful_koji_builds') {
```

Comment:
How about just performing the following? That way it fixes other instances of the use of Koji as well.
```javascript
property = property.replace('koji', 'brew')
```